docs/getting_started/tutorial.md

```diff
@@ -349,6 +349,9 @@ other page type.
 
 ![](../_static/images/tutorial/tutorial_5.png)
 
+Make sure you specify 'PUBLISH' when making new blog posts instead of
+'SAVE DRAFT' for this tutorial at the bottom of the page.
```

Comment:
I don't think this is very clear: it sounds like the tutorial is at the bottom of the page. Perhaps instead we should be changing "Now create a few blog posts as children of `BlogIndexPage`" -> "Now create and publish a few blog posts as children of `BlogIndexPage`", as by this stage in the tutorial, they have already been asked to publish a page (the index). If any further instructions on how to publish are needed, they should probably go up with the original direction to publish the index.

Reply:
Appreciate the advice, would you say it would be better to say:
"Now create and publish a few blog posts as children of `BlogIndexPage` by specifying 'PUBLISH' when making new blog posts instead of 'SAVE DRAFT'."?

Reply:
I would say:
- Just say "publish" here, because they've already published a page at this point in the tutorial.
- Add the guidance about "by clicking 'Publish' in the menu at the bottom of the page editor, rather than 'Save draft'" or similar to the instructions about publishing the index page, since that's the first time they're doing it.
I'd also avoid the caps, for preference - I don't think we use them elsewhere in the docs (see https://github.com/wagtail/wagtail/blob/main/docs/editor_manual/new_pages/previewing_and_submitting_for_moderation.rst for how we talk about the buttons elsewhere)

Reply:
I think the following should be fine now?
"Now create and publish a few blog posts as children of `BlogIndexPage` by switching to *Publish* at the bottom of the page editor, rather than *Save Draft*."

Reply:
Haha, sorry, I think we might be looking at slightly different sections. If you see https://github.com/wagtail/wagtail/blame/main/docs/getting_started/tutorial.md#L162, you can see that to get to this bit of the tutorial, they've already published a page. I think that's where "by selecting Publish at the bottom of the page editor, rather than Save Draft." belongs, because that's the first time they're performing the action.

Down here, I think we just need to say that the child pages need to be published, rather than created. Hope that makes sense!

Reply:
Ah I see what you mean now! I'll make the necessary changes then for you to review.

Reply:
Awesome, thank you!